study.md

```diff
@@ -43,7 +53,7 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
+with an array, length will return the number of elements in that array, including zero if there are no elements. length cannot be defined with enumerable but it wouldn't make sense. You cannot place length inside each element of the enumerable object the same way you can count the length of an array by counting each element.
```

Comment:
`length` makes sense on `Array` because arrays have a finite length.  It does not make sense for an `Enumerable` to have a `length` because it might not be finite.  For example, an enumerable that returns a digit of π on each iteration does not have a finite length.